convert-hf-to-gguf.py

```diff
@@ -2793,6 +2799,26 @@ def parse_args() -> argparse.Namespace:
         "--verbose", action="store_true",
         help="increase output verbosity",
     )
+    parser.add_argument(
+        "--split", action="store_true",
+        help="split the converted model into multiple files",
```

Comment:
Shouldn't this be implied by `--split-max-size` and/or `--split-max-tensors`? Otherwise nothing special happens when only using `--split-max-size`, and this can be confusing, especially since `--split` alone can't be used.

Reply:
Yes, that happened to me a few times in testing actually. Give me some time to get back to my dev environment.